src/js/views/globals/add-patient/add-patient_views.js

```diff
@@ -0,0 +1,298 @@
+import hbs from 'handlebars-inline-precompile';
+import { View } from 'marionette';
+
+import Datepicker from 'js/components/datepicker';
+import Droplist from 'js/components/droplist';
+import GroupsManagerComponent from 'js/components/groups-manager';
+
+import InputWatcherBehavior from 'js/behaviors/input-watcher';
+
+import keyCodes from 'js/utils/formatting/key-codes';
+import removeNewline from 'js/utils/formatting/remove-newline';
+import trim from 'js/utils/formatting/trim';
+
+import AddPatientTemplate from './add-patient.hbs';
+
+import './add-patient.scss';
+
+const { ENTER_KEY } = keyCodes;
+
+const SaveView = View.extend({
+  className: 'add-patient__save',
+  template: hbs`
+    <button class="button--green js-save" {{#if isDisabled}}disabled{{/if}}>{{ @intl.globals.addPatient.addPatientViews.saveView.saveBtn }}</button>
+    <button class="button--text u-margin--r-4 js-cancel">{{ @intl.globals.addPatient.addPatientViews.saveView.cancelBtn }}</button>
+  `,
+  triggers: {
+    'click .js-cancel': 'cancel',
+    'click .js-save': 'save',
+  },
+  templateContext() {
+    return {
+      isDisabled: !this.model.isValid() || this.isDisabled,
+    };
+  },
+  onSave() {
+    this.isDisabled = true;
+    this.render();
+  },
+});
+
+const SexDroplist = Droplist.extend({
+  initialize() {
+    this.collection = new Backbone.Collection([
+      {
+        text: 'Male',
+        value: 'm',
+      },
+      {
+        text: 'Female',
+        value: 'f',
+      },
+      {
+        text: 'Other',
+        value: 'other',
+      },
+    ]);
+  },
+  viewOptions: {
+    className: 'button-secondary',
+    template: hbs`{{far "user"}}{{ text }}{{#unless text}}{{ @intl.globals.addPatient.addPatientViews.sexDroplist.defaultText }}{{/unless}}`,
+  },
+});
+
+const BirthdateView = View.extend({
+  tagName: 'button',
+  template: hbs`
+    <button class="button-secondary js-date {{#if hasError}}has-error{{/if}}">
+      {{far "calendar-alt"}}{{formatDateTime birth_date "LONG" defaultHtml=(intlGet "globals.addPatient.addPatientViews.birthDateView.defaultText")}}
+    </button>
+  `,
+  templateContext() {
+    const state = this.getOption('state');
+
+    return {
+      hasError: state.get('errorField') === 'birth_date',
+    };
+  },
+  ui: {
+    'date': '.js-date',
+  },
+  triggers: {
+    'click @ui.date': 'click',
+  },
+  modelEvents: {
+    'change:birth_date': 'render',
+  },
+  initialize() {
+    this.listenTo(this.getOption('state'), {
+      'change:errorField': this.render,
+    });
+  },
+  onClick() {
+    const datepicker = new Datepicker({
+      ui: this.ui.date,
+      uiView: this,
+      state: {
+        selectedDate: this.model.get('birth_date'),
+      },
+    });
+
+    this.listenTo(datepicker, {
+      'change:selectedDate'(date) {
+        this.model.set('birth_date', date.format('YYYY-MM-DD'));
+        datepicker.destroy();
+      },
+    });
+
+    datepicker.show();
+  },
+});
+
+const FirstNameView = View.extend({
```

Comment:
I think there's an opportunity here to make an input view that has like `attr: 'first_name', shouldFocus: true, errorField, 'name'` or something.